src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({ id, sender, body, timeStamp, liked, updateLikedEntry }) => {
+  const chatEntryClassName = id % 2 === 1 ? 'chat-entry local' : 'chat-entry remote';
+
+  const updateLiked = () => {
+    updateLikedEntry(id)
```

Comment:
I'd recommend calling `updateLikedEntry()` directly rather than wrapping it in another function that gets called by the click event. It's safe to set a callback function as an event handler.